presto-hive/src/test/java/com/facebook/presto/hive/TestHiveUtil.java

```diff
@@ -156,9 +156,6 @@ private static void assertToPartitionNamesAndValues(String partitionName, List<S
 
         for (int index = 0; index < expectedKeyList.size(); index++) {
             String key = expectedKeyList.get(index);
-            if (!actual.containsKey(key)) {
```

Comment:
If the actual map does not contains a key, it should be a fail in the checkstate (line 162: "Actual result does not contains the key"). By escaping from the for loop, it will not report the failed checkstate, thus result in a passed test case.